src/bootstrap/sanity.rs

```diff
@@ -194,9 +194,7 @@ pub fn check(build: &mut Build) {
 
         if target.contains("-none-") || target.contains("nvptx") {
             if build.no_std(*target).is_none() {
```

Comment:
This function looks potentially relevant; is there a reason it's not receiving updates or so?

Reply:
Ah, I should replace the calls to `Default::default` with a new method that defaults based on the triple. Then this will work as I intended: the defaulting logic happens in one place.